bedrock/firefox/templates/firefox/desktop/desktop-base.html

```diff
@@ -1,37 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
- # License, v. 2.0. If a copy of the MPL was not distributed with this
- # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
-
-{% extends "firefox/base-resp.html" %}
-
-{% block page_title_prefix %}{% endblock %}
-{% block page_title %}{% endblock %}
-
-{% block body_id %}firefox-desktop{% endblock %}
-{% block body_class %}firefox-desktop{% endblock %}
-
-{% block extrahead %}
-  <script src="{{ static('js/libs/modernizr.custom.cssanimations.js') }}"></script>
```

Comment:
Can this JS file be removed? Seems like we can just use `@supports` these days?

Reply:
Ah, good catch. Looks like there's another orphaned Modernizr file in there, too.